x-pack/plugins/index_management/server/routes/api/data_streams/register_get_route.ts

```diff
@@ -68,9 +68,9 @@ const enhanceDataStreams = ({
   });
 };
 
-const getDataStreams = (client: ElasticsearchClient, name = '*') => {
+const getDataStreams = (client: IScopedClusterClient, name = '*') => {
   // TODO update when elasticsearch client has update requestParams for 'indices.getDataStream'
```

Comment:
Do we need to open an issue for this?

Reply:
The method is already available in the new client https://github.com/elastic/kibana/blob/b84b51cc02e78ba76c0de289aa379b0f7144263d/x-pack/plugins/fleet/server/services/epm/elasticsearch/template/template.ts#L498

Reply:
Also fixed.